packages/firebase_analytics/lib/firebase_analytics.dart

```diff
@@ -24,7 +24,7 @@ class FirebaseAnalytics {
   /// Example:
   ///
   ///     FirebaseAnalytics analytics = FirebaseAnalytics();
-  ///     analytics.android?.setMinimumSessionDuration(200000);
+  ///     analytics.android?.setAnalyticsCollectionEnabled(true);
```

Comment:
It might be less confusing to use `setSessionTimeoutDuration` since `setAnalyticsCollectionEnabled` is on the main class.

Also, since we're doing a breaking change here it might make sense to remove `setAnalyticsCollectionEnabled` from the `FirebaseAnalyticsAndroid`